dashboard/app/models/sections/section.rb

```diff
@@ -234,7 +234,17 @@ def ensure_owner_is_active_instructor
 
     si = SectionInstructor.with_deleted.find_by(instructor: user, section_id: id)
     if si.blank?
-      SectionInstructor.create!(section_id: id, instructor: user, status: :active)
+      # Using insert instead of create saves on queries to re-read the section
+      # and user from the DB to validate these are still actual objects.
```

Comment:
Is this the ideal long-term implementation? Are there other consequences of using the lower-level `insert`? Totally makes sense for the backfill task.

Reply:
Long-term, after the backfill, this insert _should_ only be running when a new section is created. The only way I could see `insert` going badly compared to `create` is if the user somehow managed to delete their account in the fraction of a second between hitting the "create section" button and that section being saved to the database.

Reply:
Is the performance optimization worth the absence of callbacks and validations running here?

Reply:
> Is the performance optimization worth the absence of callbacks and validations running here?

What do you think?

The validations query the database to make sure that the `Section` and `User` we just queried less than a second ago still exist, before we can write a `SectionInstructor` that references them. Skipping those validations will save about 8 million queries during this backfill, and the downside is that we risk storing a dangling `SectionInstructor` row in the database for every user who happens to delete their account at the exact instant this backfill job tries to fix up one of their sections. What do you think the right trade-off is here?

Reply:
The other, more long-term downside I see is someone expecting callbacks to be run and then them not being run (I may or may not have been burned by this in the past...). That said, `SectionInstructor` is a pretty small model and I'm having trouble thinking of too many validations/callbacks that would need to be added.

For the backfill, this changes makes sense to me

Reply:
I see your point about this perhaps causing a hook-related bug down the line. I'll plan to switch it back after running the backfill.

Any other concerns blocking this PR being merged?